common/calibration-model.cpp

```diff
@@ -299,6 +299,8 @@ void calibration_model::update(ux_window& window, std::string& error_message)
                     dev.as<rs2::auto_calibrated_device>().reset_to_factory_calibration();
                     _calibration = dev.as<rs2::auto_calibrated_device>().get_calibration_table();
                     _original = _calibration;
+                    table = (librealsense::ds::coefficients_table*)_calibration.data();
```

Comment:
Please avoid the c-style casting, and use cpp style (static_cast or reinterpret_cast)

Reply:
Done